app/src/sharedTest/java/org/oppia/android/app/onboarding/OnboardingFragmentTest.kt

```diff
@@ -535,6 +536,14 @@ class OnboardingFragmentTest {
     }
   }
 
+  @Test
+  fun testnextArrowDescription() {
+    launch(OnboardingActivity::class.java).use {
+      val nextArrowDescription = onView(withId(R.id.onboarding_fragment_next_image_view))
+      nextArrowDescription.check(matches(ViewMatchers.withContentDescription("Next")))
```

Comment:
Suggest writing these two lines like this
```
      onView(withId(R.id.onboarding_fragment_next_image_view)).check(
        matches(
          withContentDescription(
            "Next"
          )
        )
      )
    }
```

Reply:
Done